test_conformance/images/kernel_read_write/test_cl_ext_image_from_buffer.cpp

```diff
@@ -139,14 +139,14 @@ int image2d_from_buffer_positive(cl_device_id device, cl_context context,
                 const size_t element_size =
                     get_format_size(context, &format, imageType, flag);
 
-                /*  Alignements in pixels vs bytes */
+                /*  Alignments in pixels vs bytes */
                 if (base_address_alignment
-                    > base_address_alignment_2d * element_size)
+                    < base_address_alignment_2d * element_size)
                 {
                     test_fail("Unexpected base_address_alignment\n");
                 }
 
-                if (row_pitch_alignment > row_pitch_alignment_2d * element_size)
+                if (row_pitch_alignment < row_pitch_alignment_2d * element_size)
                 {
                     test_fail("Unexpected row_pitch_alignment\n");
                 }
```

Comment:
Can you say a little more what these conditions are designed to check for?

As far as I can tell these are checking the values queried for `CL_IMAGE_REQUIREMENTS_BASE_ADDRESS_ALIGNMENT_EXT` and `CL_IMAGE_REQUIREMENTS_ROW_PITCH_ALIGNMENT_EXT` for all image types (meaning: `1D`, `2D`, `3D`, etc.), then checking against the values queried for `CL_DEVICE_IMAGE_BASE_ADDRESS_ALIGNMENT` and `CL_DEVICE_IMAGE_PITCH_ALIGNMENT`, which are implicitly for 2D images.  Are these two values comparable in the general case?  Or can we only compare the values queried for 2D images?

Reply:
That's a good question. Let me discuss this internally, because I wasn't involved on the implementation, so I may be missing context.

Reply:
I'm going to take off "focused review" while this is under discussion.  Feel free to add it back when this is ready to go again - thanks!